packages/terra-doc-template/src/syntax-highlighter-theme.js

```diff
@@ -0,0 +1,178 @@
+const prismColors = {
```

Comment:
This file is based on https://github.com/reactjs/reactjs.org/blob/master/src/prism-styles.js

Reply:
want to add a comment that references that?

Reply:
want to add a comment that references that?

Reply:
Updated: https://github.com/cerner/terra-core/pull/1981/commits/5a0f163094ab4f15c45e48c6a99ae96ce9c37f1b